src/app/about/about.component.html

```diff
@@ -0,0 +1,136 @@
+<h3>About blockLAB</h3>
+<hr>
+<div>
+    blockLAB Stuttgart is the network for blockchain technology in the Stuttgart region. The starting point of our
+    cooperation is the mission statement of the "Blockchain Laboratory", a place for investigations and experiments
+    around blockchain technology. <br>
+
+    We believe that Blockchain technology will initiate a fundamental change in the Internet, which will have an
+    impact
+    on numerous industries and areas of life. We are convinced that the Stuttgart region with its industrial
+    economic
+    structure needs a blockchain network in order to be competitive in this key technology and to take advantage of
+    the
+    value creation opportunities that present themselves. <br>
+
+    <p>&nbsp;</p>
+
+    As the region's network for blockchain technology, we prepare information and make it available to interested
+    parties. We inform and qualify topics on blockchain technology and network specialists and companies. In
+    addition,
+    we test manufacturer-independent and neutral existing offers, concepts and technologies and develop our own
+    product
+    and service ideas. <br>
+
+    <p>&nbsp;</p>
+    Our focus is on programming, business models and the Internet of Things (IoT).
+
+</div>
+
+<p>&nbsp;</p>
+
+<h4>GitHub</h4>
+
+<a href="https://github.com/blocklab/ethverein">Find the Github Code for this Aplication here</a>
+
+<p>&nbsp;</p>
+
+<h4>Etherscan</h4>
+<a href="https://ropsten.etherscan.io/address/0x22ad32b66fACCB0b508d26B1FA57DA42a1DA9Db5#code">Find the Ethereum
```

Comment:
Shouldn't be hard-coded, we already have configuration properties `memberContractAddress` and `blockExplorerBaseUrl` for that.